student.rb

```diff
@@ -0,0 +1,12 @@
+require './person'
+
+class Student < Person
+  def initialize(classroom)
```

Comment:
While initializing the student, it should take all the parameters which a `Person` takes in addition to `classroom` parameter. You are passing `age` which is not defined anywhere along with other parameters you are passing into `super`. you will need to pass these parameters into the `initialize` method to pass them into the `super`.